### DISCUSSION TITLE
What happened to the Remix example?

### DISCUSSION THREAD

**Discussion by coindegen** [wevm/wagmi#2896] [General]
Hello, I see that framework examples have moved from here (https://github.com/wagmi-dev/wagmi/tree/main/examples ) to here (https://github.com/wagmi-dev/create-wagmi/tree/main/templates).

However I can't seem to find the Remix.run example in either location, I see only Next and Vite (react.js).

I know for a fact it was being worked on, here is the last ill-omened PR I could find from @Nikitulb :
https://github.com/wagmi-dev/wagmi/pull/666

I can't find any example on Remix's official example repo either:
https://github.com/remix-run/examples

Was it retired?

**Comment by emrea-works** [comment]
up

**Comment by pheuberger** [comment]
Would like to know too. Does that imply remix isn't well suited for web3 development?
